nemo/collections/asr/parts/mixins/mixins.py

```diff
@@ -102,11 +101,11 @@ def _setup_tokenizer(self, tokenizer_cfg: DictConfig):
                 # fallback case for older checkpoints that did not preserve the tokenizer.vocab
                 self.spe_vocab_path = None
 
-            vocabulary = {'<unk>': 0}
-            with open(vocab_path) as f:
-                for i, piece in enumerate(f):
-                    piece = piece.replace('\n', '')
-                    vocabulary[piece] = i + 1
+            vocabulary = {}
```

Comment:
Does it work with all types of the tokenizers?

Reply:
All SPE type tokenizers yes. WPE has its vocab file so different handling